http/pom.xml

```diff
@@ -41,19 +41,9 @@
                 <version>${kafka.version}</version>
             </dependency>
             <dependency>
-                <groupId>io.jaegertracing</groupId>
-                <artifactId>jaeger-client</artifactId>
-                <version>${jaeger.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.opentracing.contrib</groupId>
-                <artifactId>opentracing-kafka-client</artifactId>
-                <version>${opentracing-kafka.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.opentracing.contrib</groupId>
-                <artifactId>opentracing-kafka-streams</artifactId>
-                <version>${opentracing-kafka.version}</version>
+                <groupId>io.strimzi</groupId>
+                <artifactId>test-client-tracing</artifactId>
+                <version>${project.version}</version>
```

Comment:
Is the version needed?

Reply:
This is how it was done before -- all deps in http/pom.xml have versions.

Reply:
So it should go into each submodule?